environment.yml

```
@@ -1,26 +1,36 @@
 name: gui
+channels:
+    - conda-forge
+    - mantid/label/nightly
+    - mantid
+    - anaconda
 dependencies:
-- cycler=0.10.0=py27_0
-- freetype=2.5.5=0
-- libpng=1.6.17=0
-- matplotlib=1.5.1=np111py27_0
-- mkl=11.3.1=0
-- numpy=1.11.0=py27_0
-- openssl=1.0.2g=0
-- pip=8.1.1=py27_1
-- pyparsing=2.1.1=py27_0
-- pyqt=5
-- python=2.7.11=0
-- python-dateutil=2.5.2=py27_0
-- pytz=2016.3=py27_0
-- qt=5
-- readline=6.2=2
-- setuptools=20.7.0=py27_0
-- sip=4.16.9=py27_0
-- six=1.10.0=py27_0
-- sqlite=3.9.2=0
-- tk=8.5.18=0
-- wheel=0.29.0=py27_0
-- zlib=1.2.8=0
+- cycler
+- freetype
+- libpng
+- matplotlib
+- mkl
+- numpy=
+- openssl
+- pip
+- pyparsing
+- pyqt>=5.12.3,<6
+- python=3.8.*
+- python-dateutil
+- pytz
+- qt>=5.12.9,<6
+- qtpy>=1.9.0
+- readline
+- setuptools
+- sip
+- six
+- sqlite
+- tk
+- wheel
+- zlib
 - pillow
+- scipy
+- h5py>=2.10.0,<3
+- hdf5=1.10.*
+- mantid==6.3
```

```suggestion
- mantid>=6.3,<6.4
```
is another way to pin the minor version